### ISSUE TITLE
Tests

### ISSUE THREAD

**Issue by sjlagasse** [jetmech/expresso#6]
I'm a little confused here. Usually there has always been 89 tests and your registered 198 with one failing. Did you build in some of your own tests? If so, very cool!

**Comment by jetmech** [comment]
Yeah, I wrote a bunch of tests for the classes I put together to ensure that they manipulated the database as they were intended to.

That way if a route test failed but the class test passed then I knew where to look for the :bug:.

I also used the tests to ensure that errors would be thrown if any properties were not set correctly. I tried to separate the routing logic and database logic to make the code more maintainable.

I'm curious to know which test failed though! They all passed on my machine. :confused:
